specutils/io/asdf/tags/spectra.py

```diff
@@ -9,7 +9,8 @@
                            tagged_tree_to_custom_tree)
 from astropy.io.misc.asdf.tags.unit.unit import UnitType
 
-from ....spectra import SpectralCoord, Spectrum1D, SpectrumList
+from ....extern.spectralcoord import SpectralCoord
```

Comment:
Isn't there going to need to be a `SpectralCoordType` in astropy to get this to work, too? (and then that means it should be imported from there instead?)

Reply:
Yes good point, will fix that

Reply:
So I think the correct approach is to just not define the Type class when astropy 4.1 is available since the ASDF tag and schemas are defined in astropy core for the class we then use in specutils. I'll fix that.